docs/pages/x/api/date-pickers/pickers-action-bar.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import ApiPage from 'docsx/src/modules/components/ApiPage';
+import mapApiPageTranslations from 'docs/src/modules/utils/mapApiPageTranslations';
+import jsonPageContent from './pickers-action-bar.json';
+
+export default function Page(props) {
+  const { descriptions, pageContent } = props;
+  return <ApiPage descriptions={descriptions} pageContent={pageContent} />;
+}
+
+Page.getInitialProps = () => {
+  const req = require.context(
+    'docsx/translations/api-docs/date-pickers',
+    false,
+    /\/pickers-action-bar(-[a-z]{2})?\.json$/,
```

Comment:
Should we maybe add this API entry in the submenu as well as maybe the "index" page, even though it's link has already been removed, so, it's probably no longer that relevant. 🤔 
![Screenshot 2022-12-22 at 09 37 08](https://user-images.githubusercontent.com/4941090/209082244-998b1b05-3893-451e-b773-4f3c8f032e71.png)

Reply:
I first did not add them because the tables were nearly empty, but it was a bug from my side